pkg/extension/src/scripts/content/prepare-content.js

```diff
@@ -62,6 +62,28 @@
     })
   }
 
+  /**
+   * Determines whether a page is a feed
+   * Currently only feed pages of the XML family and selected feed format in it are recognized (i.e. RSS or Atom)
+   * 
+   * @returns true if it's feed page, false otherwise
+   */
+  function isFeedPage() {
+    const fileExtension = window.location.pathname.slice(-4).toLowerCase()
+    const hasXMLExtension = fileExtension === '.xml'
+    const xmlContentTypes = [
```

Comment:
I think we have a few other types we handle here /cc @sywhb

Reply:
yeah, i think `.rss` and `.html` could belong to here too

Reply:
ah the problem with HTML though is then it wont work for nomal pages, so maybe we have to just be conservative and handle XML here, even though a lot of sites return it as HTML.